app/lib/shared/integrity.dart

```diff
@@ -40,6 +40,11 @@ class IntegrityChecker {
   final DatastoreDB _db;
   final int _concurrency;
 
+  static const _knownUnmappedFields = {
```

Comment:
Please document the scope of this (is it only about packages?)

Reply:
renamed + done.